test/jdk/java/util/zip/DeflateIn_InflateOut.java

```diff
@@ -149,7 +152,9 @@ private static void SkipBytes() throws Throwable {
         int numReadable = 0;
 
         // Count number of bytes that are read
-        reset();
+
+		// Reset Input data and Streams
```

Comment:
Not sure why this one got a comment when the others that changed from reset() to resetAllI() didn't. If this comment is retained, spaces should be used for indenting rather than tabs.

Reply:
I'm fine if this comment is just removed.

Reply:
Removed comment in https://github.com/ibmruntimes/openj9-openjdk-jdk11/compare/6d92003cfe8e2e8d76535edcd9d18fcd9aa4b70c..ee1497e7a2ee0a56a59c89136d15b5adf9d41c73